### PR TITLE
fix: bugs 2946, 2951

### PR DESCRIPTION
**Pull request by kirgene**
* fix #2946
* fix #2951

### CODE REVIEW

File: modules/outbound-domain-event-handler/src/application/handlers/bulk_transfers_callback_received.ts [hunk 1]
```diff
@@ -19,8 +19,8 @@ export async function handleBulkTransfersCallbackReceived(
             bulkId: bulkTransfersCallbackReceivedMessage.getKey(),
             content: {
                 batchId: bulkTransfersCallbackReceivedMessage.batchId,
-                bulkTransferId: bulkTransfersCallbackReceivedMessage.bulkTransferId,
```
Comment (mdebarros) [inline modules/outbound-domain-event-handler/src/application/handlers/bulk_transfers_callback_received.ts:22]: You are going to need this in the `bulkTransfersErrorResult` <-- it does not contain the bulkTransferId, or am I missing something?

File: modules/private-shared-lib/src/domain/individual_transfer_entity.ts [hunk 2]
```diff
@@ -53,18 +54,32 @@ export enum IndividualTransferInternalState {
     TRANSFERS_SUCCESS = 'TRANSFERS_SUCCESS',
 }
 
+export interface IndividualQuoteResponse extends SDKSchemeAdapter.V2_0_0.Outbound.Types.individualQuoteResult {
+    expiration: SDKSchemeAdapter.V2_0_0.Outbound.Types.DateTime;
```
Comment (mdebarros) [inline modules/private-shared-lib/src/domain/individual_transfer_entity.ts:58]: I believe this missing quote a few fields?

e.g. ilpCondition, etc?

File: modules/private-shared-lib/src/domain/individual_transfer_entity.ts [hunk 3]
```diff
@@ -53,18 +54,32 @@ export enum IndividualTransferInternalState {
...
+}
+
+export interface IndividualTransferResponse extends SDKSchemeAdapter.V2_0_0.Outbound.Types.individualTransferResult {
+    completedTimestamp?: SDKSchemeAdapter.V2_0_0.Outbound.Types.DateTime;
```
Comment (mdebarros) [inline modules/private-shared-lib/src/domain/individual_transfer_entity.ts:62]: This is also missing some fields, i.e. `ilpFulfilment`, etc?

File: modules/outbound-domain-event-handler/src/application/handlers/bulk_transfers_callback_received.ts (hunk 1 above)
Comment (kirgene) [reply modules/outbound-domain-event-handler/src/application/handlers/bulk_transfers_callback_received.ts:22]: `bulkTransferId` is part of a `bulkTransfersResult` (`BulkTransferResponse` type)

File: modules/private-shared-lib/src/domain/individual_transfer_entity.ts (hunk 2 above)
Comment (kirgene) [reply modules/private-shared-lib/src/domain/individual_transfer_entity.ts:58]: well, this is what we get from the switch:
https://github.com/mojaloop/api-snippets/blob/f4dfd101d68c6a55e71867d167dc765b16129214/docs/fspiop-rest-v1.1-openapi3-snippets.yaml#L1862

https://github.com/mojaloop/api-snippets/blob/f4dfd101d68c6a55e71867d167dc765b16129214/docs/fspiop-rest-v1.1-openapi3-snippets.yaml#L3500

https://github.com/mojaloop/api-snippets/blob/f4dfd101d68c6a55e71867d167dc765b16129214/docs/fspiop-rest-v1.1-openapi3-snippets.yaml#L3473

File: modules/private-shared-lib/src/domain/individual_transfer_entity.ts (hunk 3 above)
Comment (kirgene) [reply modules/private-shared-lib/src/domain/individual_transfer_entity.ts:62]: similar to the above comment:
https://github.com/mojaloop/api-snippets/blob/f4dfd101d68c6a55e71867d167dc765b16129214/docs/fspiop-rest-v1.1-openapi3-snippets.yaml#L3622

and
https://github.com/mojaloop/api-snippets/blob/master/sdk-scheme-adapter/v2_0_0/components/schemas/individualTransferResult.yaml

**Review by mdebarros — COMMENTED** [review]
+1 

Just need to resolve the CI failures.

**Review by mdebarros — APPROVED** [review]
+1

**Pull request merged**